src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,39 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
     }
```

Comment:
3. **Unit Tests:** It's important to have unit tests for each method, ideally to cover different input scenarios and edge cases. This will help in ensuring the correctness of the calculator service.